core/data/fixtures/classes.rb

```diff
@@ -0,0 +1,3 @@
+module DataClasses
```

Comment:
Should be `DataSpecs`, that will address the `Global constants leak check`

Reply:
Because https://github.com/ruby/spec/blob/58c132b1cf4c8cee238201dee3e30e8cc251bda5/.mspec.constants#L186, ideally that list would be smaller.